### PR TITLE
Build zenithdb/compute-node:latest in CI (zenithdb/console#125)

### PR DESCRIPTION
**Pull request by ololobus**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+FROM zenithdb/compute-tools:latest AS compute-deps
+# Only to get ready apply_conf binary as a dep
+
+#
+# Image with Postgres build deps
+#
+FROM debian:buster-slim AS build-deps
+
+RUN apt-get update && apt-get -yq install automake libtool build-essential bison flex libreadline-dev zlib1g-dev libxml2-dev \
+                                          libcurl4-openssl-dev
```
Comment (kelvich) [inline Dockerfile:13]: I think libcurl4 isn't necessary now

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+#
+# Image with built Postgres
+#
+FROM build-deps AS pg-build
```
[the remark this answers is not in the document]
Comment (ololobus) [reply Dockerfile:18]: I think so, to don't mess with copying (clonning) `apply_conf` sources from another repo and building it each time with `postgres` / compute node

[the remark this answers is not in the document]
Comment (ololobus) [reply Dockerfile:18]: Wow, missed the line

[the remark this answers is not in the document]
Comment (ololobus) [reply Dockerfile:18]: I don't know, just to keep dependencies and build results logically separated. Docker should put different `RUN`'s into different layers, so I don't expect any performance benefits

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+RUN mkdir /pg && chown postgres:postgres /pg
+
+# Copy source files
+COPY . /pg/
+
+# Build and install Postgres locally
+RUN mkdir /pg/compute_build && cd /pg/compute_build && \
+    ../configure CFLAGS='-O0 -g3' --prefix=$(pwd)/postgres_bin --enable-debug --enable-cassert --enable-depend && \
+    # Install main binaries and contribs
+    make MAKELEVEL=0 -j $(getconf _NPROCESSORS_ONLN) -s install && \
```
Comment (funbringer) [inline Dockerfile:31]: What's the benefit of `_NPROCESSORS_ONLN` over `nproc`?

Comment (ololobus) [reply Dockerfile:31]: There is no `nproc` on macOS and I just use `getconf _NPROCESSORS_ONLN` everywhere. Here is a linux container, so it isn't needed, but I do copy-paste some script parts to/from local scripts and this is just more convinient

**Pull request merged**